src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx

```diff
@@ -104,7 +104,7 @@ const formSchema = yup
                 is: (lowShortCircuitCurrentLimit) => lowShortCircuitCurrentLimit != null,
                 then: (schema) => schema.required(),
             }),
-        [BUS_BAR_COUNT]: yup.number().min(1).nullable().required(),
+        [BUS_BAR_COUNT]: yup.number().min(1, 'BusBarCountMustBeGreaterThanOrEqualToOne').nullable().required(),
         [SECTION_COUNT]: yup.number().min(1).nullable().required(),
```

Comment:
we could fix the translation for SECTION_COUNT also in this PR

```suggestion
        [SECTION_COUNT]: yup.number().min(1, 'SectionCountMustBeGreaterThanOrEqualToOne').nullable().required(),
```